applications/test/MediaFeatureTest.scala

```diff
@@ -32,13 +33,17 @@ import org.scalatest.{DoNotDiscover, Matchers, GivenWhenThen, FeatureSpec}
     scenario("Include Guardian byline") {
       goTo("/film/video/2013/aug/14/chloe-grace-moretz-kick-ass-2-video") { browser =>
         import browser._
-        el(".byline").text should be("Ben Child and Henry Barnes, theguardian.com")
+        el(".byline").html().toString should include("Ben Child")
+        el(".byline").html().toString should include("Henry Barnes")
+        el(".byline").html().toString should include("theguardian.com")
```

Comment:
What happened to the `text` method anyway? Surprised if it's been removed!

Reply:
Yep. In this case, `.html()` gives the HTML markup. There wasn't an obvious way to get the original `text`, so I went for testing the relevant pieces of content. Which I thought was good enough :)